Assets/Package/Runtime/Console/ConsoleManager.cs

```diff
@@ -171,13 +172,11 @@ private void RegistorMethodFromCommand(ConsoleCommand command, MethodInfo method
 
         public void AppendCommand(ConsoleCommand command)
         {
-            EnsureInitialized();
```

Comment:
Jeżeli coś doda komendę z zewnątrz przed inicjalizacją ConsoleManagera to polecą błędy. Tutaj w ogóle był ten stackoverflow?

Reply:
Tak. Ogólnie najprostszym sposobem żeby go fixnąć byłoby zostawienie ustawienia isInitialized=true na samej górze EnsureInitialized, ale to trochę nie robi sensu, bo przecież nic nie jest jeszcze wtedy zainicjalizowane :p

Reply:
Jeszcze jedno rozwiązanie jakie widzę to drugi bool, który jest true tylko w trakcie inicjalizacji. Coś w stylu isInitializing.

Reply:
Możesz dodać drugą isInitializing i skipować inicjalizację wtedy też.

Reply:
To nie będzie trochę brudnawe? Nie spotkałem się jeszcze z taką flagą.

Reply:
A czemu miałoby to być "brudnawe"?

Reply:
Dodajemy kolejną flagę inicjalizacji, żeby naprawić stack overflow. Może "haxowe" jest lepszym określeniem w tym przypadku.

Reply:
Nie widzę tutaj nic haxowego. Flaga "isInitialziing" w parze z "isInitialized" występuje bardzo często. Wcześniej w teorii nie była potrzebna ale jednak ze względu na naturę managera, który jest "lazy" powoduje, że się przyda bo podczas inicjalizacji używamy publicznego API

Reply:
W takim razie wrzuciłem fixa z wykorzystaniem isInitialzing :)